pkg/segment/writer/segwriter.go

```diff
@@ -463,12 +465,14 @@ func InitColWip(segKey string, colName string) *ColWip {
 	}
 
 	cbuf := *wipCbufPool.Get().(*[]byte)
+	dePack := *wipCbufPool.Get().(*[]byte)
```

Comment:
You're using the same pool because `cbuf` and `dePack` should require roughly the same amount of memory, right? If they use significantly different amounts of memory, we might want to have separate pools

(this could also be changed in a different PR if we find it helps performance)